test/security/security_test.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'dart:convert';
+
+import 'package:p2p_task/security/key_helper.dart';
+import 'package:p2p_task/network/messages/introduction_message.dart';
+import 'package:p2p_task/network/messages/packet.dart';
+import 'package:p2p_task/network/web_socket_peer.dart';
+
+void main() {
+  var keyHelper = KeyHelper();
+  var keyPair = keyHelper.generateRSAkeyPair();
+  var message = 'Really private stuff';
+
+  group('#security', () {
+    test('encryption with keys', () {
+      var encrypted = keyHelper.encrypt(keyPair.publicKey, message);
+      expect(message != encrypted, true);
+      var plain = keyHelper.decrypt(keyPair.privateKey, encrypted);
+
+      expect(message == plain, true);
```

Comment:
Should be possible to use `expect(message, isNot(plain))` to produce better error messages if it fails (also in some other tests).

Reply:
expect(message, equals(plain)) 👌

Reply:
Nice catch but I originally meant line 17 :smile:

Reply:
oh yes haha 😄